Library/Homebrew/test/tap_spec.rb

```diff
@@ -209,6 +209,31 @@ def setup_git_repo
       expect { already_tapped_tap.install full_clone: true }.to raise_error(TapAlreadyUnshallowError)
     end
 
+    describe "force_auto_update" do
+      before do
+        setup_git_repo
+      end
+
+      let(:already_tapped_tap) { described_class.new("Homebrew", "foo") }
+
+      it "defaults to nil" do
+        expect(already_tapped_tap).to be_installed
+        expect(already_tapped_tap.config["forceautoupdate"]).to be_nil
+      end
+
+      it "enables forced auto-updates when true" do
+        expect(already_tapped_tap).to be_installed
+        expect { already_tapped_tap.install force_auto_update: true }.not_to raise_error
```

Comment:
`.not_to raise_error` here is pretty useless, since the spec would fail anyways in that case.

Reply:
Good point. Removed.